devito/builtins.py

```diff
@@ -123,46 +135,61 @@ def fset(f, g):
         else:
             raise NotImplementedError
 
-    lw = int(_order*sigma + 0.5)
+    try:
+        dtype = f.dtype.type
```

Comment:
? this is for numpy arrays ? if so, maybe leave a comment?

Reply:
Yes. Will add comment.